docs/vc-http-api.yml

```diff
@@ -110,9 +112,13 @@ paths:
               schema:
                 $ref: "#/components/schemas/VerifyPresentationResponse"
         "400":
-          description: invalid input!
+          description: Invalid or malformed input 
+        "413":
+          description: Payload too large
```

Comment:
Yes! +1 to this -- spec should say that implementations can set payload limits.

Reply:
yep, I think this should be helpful ;)